src/main/java/io/neonbee/data/internal/DataContextImpl.java

```diff
@@ -52,6 +54,8 @@ public class DataContextImpl implements DataContext {
 
     private static final Pattern BEARER_AUTHENTICATION_PATTERN = Pattern.compile("Bearer\\s(.+)");
 
+    private static final String RESPONSE_METADATA_KEY = "responsedata";
+
```

Comment:
you could put this one above to the other KEY constants also here it still says Metadata, so suggestion would be:

private static final String RESPONSE_DATA_KEY = "responseData";

Reply:
Done